internal/text/text.go

```diff
@@ -0,0 +1,74 @@
+package text
+
+import (
+	"fmt"
+	"net/url"
+	"regexp"
+	"strings"
+	"time"
+
+	"github.com/cli/go-gh/pkg/text"
+	"golang.org/x/text/cases"
+	"golang.org/x/text/language"
+)
+
+var whitespaceRE = regexp.MustCompile(`\s+`)
+
+func Indent(s, indent string) string {
+	return text.Indent(s, indent)
+}
+
+// Title returns a copy of the string s with all Unicode letters that begin words mapped to their Unicode title case.
+func Title(s string) string {
+	c := cases.Title(language.English)
+	return c.String(s)
+}
+
+// RemoveExcessiveWhitespace returns a copy of the string s with excessive whitespace removed.
+func RemoveExcessiveWhitespace(s string) string {
+	return whitespaceRE.ReplaceAllString(strings.TrimSpace(s), " ")
+}
+
+func DisplayWidth(s string) int {
+	return text.DisplayWidth(s)
+}
+
+func Truncate(maxWidth int, s string) string {
+	return text.Truncate(maxWidth, s)
+}
+
+func Pluralize(num int, thing string) string {
+	return text.Pluralize(num, thing)
+}
+
+func FuzzyAgo(a, b time.Time) string {
```

Comment:
`FuzzyAgo` used to take in a `time.Duration` argument but I changed it to take in two `time.Time` arguments to match `FuzzyAgoAbbr`. Also the call sites were using `time.Since` or `time.Sub` to create the `time.Duration` argument so that is now simplified and taken care of by the function.